eslint.config.mjs

```diff
@@ -27,17 +27,18 @@ export default fixupConfigRules([
       'plugin:@typescript-eslint/recommended-requiring-type-checking',
       'plugin:react/recommended',
       'plugin:jest-react/recommended',
+      // 'plugin:storybook/recommended',
```

Comment:
Leaving this comment in here because we eventually want to enable this one, but it's broken right now.